docker/pypi/wmagent/bin/manage-common.sh

```diff
@@ -317,7 +317,7 @@ _renew_proxy(){
                 echo "$FUNCNAME: ERROR: Failed to renew expired myproxy"; return $(false) ;}
 
         # Stay safe and always change the service {cert,key} and myproxy mode here:
-        sudo chmod 400 $WMA_CERTS_DIR/*
+        chmod 400 $WMA_CERTS_DIR/*
```

Comment:
good catch... this one I've forgot to remove long ago, and was driving me crazy for so long, to the point I was eager for all other changes to finish so that I can create a separate issue just to fix it.